scene/gui/link_button.cpp

```diff
@@ -140,6 +140,6 @@ void LinkButton::_bind_methods() {
 
 LinkButton::LinkButton() {
 	underline_mode = UNDERLINE_MODE_ALWAYS;
-	set_enabled_focus_mode(FOCUS_NONE);
+	set_focus_mode(FOCUS_NONE);
```

Comment:
Same comment as for the `master` PR, should not be necessary as it's the default.